docs/en/observability/deploy-agent-to-send-data.asciidoc

```diff
@@ -7,6 +7,8 @@ scenes, {agent} it runs the {beats} shippers or {elastic-endpoint} required for
 your configuration. The shippers are managed by {agent} -- you do not install or
 configure them individually.
 
+//REVIEWERS: Is Elastic Endpoint correct here? ^^
```

Comment:
There's an extra word in line 6 after {agent} - `it` can be removed.

Reply:
@dedemorton so `{elastic-endpoint}` is the correct attribute :) I would recommend some slight modifications to the second half of the sentence though. In its current state, it almost looks as though we're telling users that Elastic Endpoint performs similar functions to Beats and the phrase "required for your configuration" seems a tad disconnected from what Endpoint does. The way I understand it, Agent can control and interact with Elastic Endpoint and Endpoint uses functionality that the Security solution offers such as policy configuration, trusted applications, alert exceptions, and more to monitor and report system activity. 

How about revising the sentence to something like:
You deploy the agent to your hosts or containers, and behind the scenes, {agent} runs {beats} shippers to gather data or {elastic-endpoint} to monitor and report system activity.

@jmikell821 @joepeeples your thoughts are welcome here as well! :)

Reply:
I like your revision, but I am going to push an alternate description because we are trying to converge on a single description of Elastic Agent that is more inclusive of its capabilities, including the ability to collect and forward data from remote hosts and hardware.